File: python/bc2543.py
while True:
    count = 0

    try:
        fLi = input().split(" ")

        nCa = int(fLi[0])
        ind = int(fLi[1])

        for c in range(0, nCa):
            sLi = input().split(" ")

            fVal = int(sLi[0])
            sVal = int(sLi[1])

            if fVal == ind and sVal == 0: count += 1

        print(count)

    except EOFError:
        break

File: cpp/bc2543.cpp
#include <iostream>

using namespace std;

int main()
{
    int count, nCa, ind, fVal, sVal;
    
    while(cin >> nCa >> ind)
    {
        count = 0;
        
        for(int i = 0; i < nCa; i++)
        {
            cin >> fVal >> sVal;
            if(fVal == ind && sVal == 0) count++;
        }
        cout << count << endl;
    }
}
